testing/scenario_app/bin/run_ios_tests.dart

```diff
@@ -409,11 +409,12 @@ String _zipAndStoreFailedTestResults({
       '-q',
       '-r',
       outputPath,
-      resultBundlePath,
+      resultBundle.path,
     ],
   );
   if (result.exitCode != 0) {
     throw Exception('Failed to zip the test results: ${result.stderr}');
   }
+  resultBundle.deleteSync(recursive: true);
```

Comment:
@matanlurey taking a crack at fixing this https://github.com/flutter/engine/pull/53717/commits/99a2e4e26fcd7f88919e1fa3e537d4cc2d119943--I think the difference is that the results aren't deleted between runs.

Reply:
Close, it was the `dumpDir` above leaking. PTAL!